solutions/swb-reference/README.md

```diff
@@ -68,11 +68,237 @@ This step is necessary to setup Service Catalog portfolio and products
 
 ## Integration Tests
 
+`swb-reference` package contains integration tests that run API tests against SWB APIs, they can be configured to run automatically as part of a GitHub workflow or CI/CD pipeline.
+
+We recommend to create a non-production environment with a different <STAGE> name for integration test as this process will create resources in the environment they are executed against.
```

Comment:
I think I meant that we recommend to create a _project_ within SWB that is intended only for integration tests. Users do not need an entirely new deployment of SWB for testing (although they may choose to). The rec that we gave in V1 was to create a project called "IntegTestProject" or something and use that for integration tests. That would eliminate the steps in this readme that say list the projects and choose one. They would instead say "create a new project for integ tests" and include that information in the config for the tests/

Reply:
Updated readme to display in details pre requisites